templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,42 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="title"></h2>
+            <P id="facet-panels"></P>
+        </div>
+        <!-- Fetching facet knowledge panel -->
+        <script>
+            let facet_kp_url = "https://facets-kp.openfoodfacts.net/render-to-html?";
+            let params = "facet_tag=[% tagtype_name %]&value_tag=[% tagid %]&sec_facet_tag=[% tagtype2_name %]&sec_value_tag=[% tagid2 %]";
+            // checking if data is filtered on country 
+            if ("[% country_code %]" !== "world"){
+                // Adding language code to get transalted data from facet-knowledge-panel API
+                // Removing langcode from county name l.e, "en:france" to "france"
+                country_name = "[% country %]".slice(3);
+                params += "&lang_code=[% country_code %]&country=" + country_name;
+            }
+            fetch( fecet_kp_url + params )
+            .then((response) => {
+                if (response.ok) {
+                    return response.text();}
+                else {
+                    throw new Error("NETWORK RESPONSE ERROR");
```

Comment:
Maybe something a bit more specific? e.g. mention it's the facets-panels that have a network issue, and log which one.